docs/references/general/rawacf.md

```diff
@@ -1,3 +1,110 @@
+| *noise.mean*            | *None*   | float     | Average noise from clear frequency band                                                 |
+| *channel*               | *None*   | short     | Channel number, used to denote different Tx/Rx channels on Stereo radars, and to denote changes in radar operating parameters between scans, e.g. alternating between 2 frequencies scan-to-scan      |
+| *bmnum*                 | *None*   | short     | Beam number, zero based indexing                                                        |
+| *bmazm*                 | *degrees*   | float     | Beam azimuth                                                                            |
+| *scan*                  | *None*   | short     | Scan flag, switches between 0 to +/-1. For some radars, the sign indicates scan direction. |
+| *offset*                | *us*     | short     | Offset between channels for a stereo radar                                              |
+| *rxrise*                | *us*     | short     | Receiver rise time                                                                      |
+| *intt.sc*               | *s*      | short     | Number of seconds of integration time                                                   |
+| *intt.us*               | *us*     | int       | Number of microseconds of integration time                                              |
+| *txpl*                  | *us*     | short     | Transmitted pulse length                                                                |
+| *mpinc*                 | *us*     | short     | Multi-pulse increment                                                                   |
+| *mppul*                 | *None*   | short     | Number of pulses in the sequence                                                        |
+| *mplgs*                 | *None*   | short     | Number of lags estimated from the sequence                                              |
+| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in RAWACF file format. | 
+| *ifmode*                | *None*   | short     | A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of mixing to baseband and then digitizing (`ifmode=0`).         |
+| *nrang*                 | *None*   | short     | Number of range gates                                                             |
+| *frang*                 | *km*     | short     | Distance to the first range gate                                                        |
+| *rsep*                  | *km*     | short     | Range seperation                                                                        |
+| *xcf*                   | *None*   | short     | XCF flag, indicates if XCF for elevation angle estimation is available (1) or not (0)   |
```

Comment:
The XCF flag indicates more than just the availability of elevation angle measurements (although to be fair most people tend not to use the velocity / power / width / etc derived from the XCF).

Reply:
Is there a suggestion here? Please use the suggestion box to add extra information. Thank you!

Reply:
I guess there's the possibly that the non-zero xcf value represents how many scans between elevation angle collection?  Is that what you were thinking about @egthomas?  For example, an xcf of 4 would mean elevation angle is collected on every 4th scan.  I can add a suggestion about that if that's what you were thinking @egthomas.

Reply:
@ksterne you're referring to how the `xcf` option on a radar schedule line determines how frequently XCF data is recorded; this information is generally stored in an `xcnt` variable within the control program and is kept separate from the `xcf` field of the radar parameter block.  I meant that the XCF flag indicates the presence of the full range of fitted parameters from the cross-correlation (velocity, power, width, phi0, their associated errors, etc) - not just the presence of elevation angles, as the documentation currently suggests.

Reply:
```suggestion
| *xcf*                   | *None*   | short     | XCF flag, indicating whether the power, velocity, spectral width and phi0 parameters fitted from the cross-correlation function are available (xcf=1) or not available (xcf=0)   |
```

Reply:
Thanks for the clarification @egthomas .  I was a little suspicious that something like the xcf count (`xcnt`) from the control program world would make it into the rawacf file, but wouldn't have thought twice about it.  Would you like to commit @ecbland's suggestion?

Reply:
@ecbland I think he is looking for a suggestion? Or do you want to commit it?

Reply:
@mts299 I think my suggestion is fine here, as it addresses the original comment from @egthomas